composer.json

```diff
@@ -54,7 +58,7 @@
   "prefer-stable": true,
   "scripts": {
     "phpcbf": "phpcbf .",
-    "phpcs": "phpcs .",
+    "phpcs": "phpcs . -sn",
```

Comment:
We shouldn't define these flags here. The `s` flag is already defined in `.phpcs.xml` and we don't want to suppress warnings.